static/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -545,8 +588,15 @@ class GroupDetails extends Component<Props, State> {
   }
 
   renderPageContent() {
-    const {error: isError, group, project, loading} = this.state;
-    const isLoading = loading || (!group && !isError);
+    const {
+      error: isError,
+      group,
+      project,
+      loading,
+      loadingReplaysCount: isLoadingreplaysCount,
+      replaysCount,
+    } = this.state;
+    const isLoading = loading || isLoadingreplaysCount || (!group && !isError);
```

Comment:
This is going to block the entire view from loading while it waits for the replays count. Instead we should just show an empty string, or a space ` ` if the initial count value is null.